.gitignore

```diff
@@ -62,3 +62,4 @@ data
 
 # ignore test properties
 core/src/test/resources/tidb_config.properties
+/core/src/test/resources/tispark-test/
```

Comment:
Why should we ignore test here?

Reply:
Because tispark-test is a git module pulled from `https://github.com/pingcap/tispark-test-data.git`. 
If we include it in our project, it may cause problems when tispark-test.git is modified, e.g., when its base repository is changed. ( and it did! )

Reply:
I found this https://stackoverflow.com/questions/7912022/do-you-ignore-a-git-submodule-in-your-gitignore-or-commit-it-to-your-repo

Reply:
If you do want to ignore this directory, I think we may need to remove the first slash here.

Reply:
Actually this slash is generated... It works and I didn't add it.

Reply:
Ok, this slash is okay to ignore the dir...

Reply:
Will revert this change according to discussion.